vpr/src/route/clock_connection_builders.cpp

```diff
@@ -131,19 +132,19 @@ int RoutingToClockConnection::create_virtual_clock_network_sink_node(
  */
 
 void ClockToClockConneciton::set_from_clock_name(std::string clock_name) {
-    from_clock = clock_name;
+    from_clock = std::move(clock_name);
```

Comment:
all these: instead of using std::move from a by-value parameter, we should use assignment and pass in a `const std::string &`